extractor/main.go

```diff
@@ -247,6 +247,9 @@ func extractAddMessages(filePath string) ([]vm.MsgAddPackage, error) {
 			    //TODO: throw error
 			}
 
+			if msgAddPkg.Package == nil {
+				// TODO: throw error
```

Comment:
Come on! The TODOS are the parts you have to fill :rofl:

Reply:
I did, in the final commit :) 
[928801c](https://github.com/gnolang/tx-exports/pull/5/commits/928801c40dc6d206670a6f48d388eb1012b481ad)

Reply:
ok, they are handled in a following commit :+1: